ibeis/web/test_api.py

```diff
@@ -6,6 +6,9 @@
 import hmac
 import requests
 
+import utool as ut
```

Comment:
Please install PIP8 for linting.  Please refer to here (https://github.com/WildbookOrg/ibeis_plugin_identification_example/blob/0ea784a8f723e31520fbc5423f53882db0d752e3/ibeis_plugin_identification_example/_plugin.py#L362) for the correct linting options to ignore.

Reply:
Also, listed here more explicitly: https://github.com/WildbookOrg/ibeis#code-style-guidelines